=== Suite started at 2024-10-16 11:52:00
=== Starting test, 4 test cases
=cases         4
=user          otptest
=host          a46a33ecc1a5
=hosts         a46a33ecc1a5 
=emulator_vsn  15.1.1
=emulator      beam
=otp_release   28
=started       2024-10-16 11:52:00
=case          config_func_error_1_SUITE:init_per_suite
=logfile       config_func_error_1_suite.init_per_suite.html
=started       2024-10-16 11:52:00
=ended         2024-10-16 11:52:00
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_iptc
=logfile       config_func_error_1_suite.exit_in_iptc.html
=started       2024-10-16 11:52:01
=== ERROR! config_func_error_1_SUITE:init_per_testcase(exit_in_iptc, [{watchdog,
                                                                   <0.3731.0>},
                                                                  {tc_logfile,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/config_func_error_1_suite.exit_in_iptc.html"},
                                                                  {tc_group_properties,
                                                                   []},
                                                                  {tc_group_path,
                                                                   []},
                                                                  {data_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                  {priv_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"}]) failed!
	Reason: {'EXIT',
                    {function_clause,
                        {ct_test_support,random_error,
                            [x],
                            [{file,"ct_test_support.erl"},{line,418}]}}}

=ended         2024-10-16 11:52:01
=result        auto_skipped: {failed,
                              {config_func_error_1_SUITE,init_per_testcase,
                               {'EXIT',
                                {function_clause,
                                 {ct_test_support,random_error,
                                  [x],
                                  [{file,"ct_test_support.erl"},
                                   {line,418}]}}}}}
=elapsed       0.000000s
=== *** SKIPPED test case 1 of 4 ***
=case          config_func_error_1_SUITE:exit_in_eptc
=logfile       config_func_error_1_suite.exit_in_eptc.html
=started       2024-10-16 11:52:01
=== WARNING! config_func_error_1_SUITE:end_per_testcase(exit_in_eptc, [{tc_status,
                                                                    ok},
                                                                   {watchdog,
                                                                    <0.3736.0>},
                                                                   {tc_logfile,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/config_func_error_1_suite.exit_in_eptc.html"},
                                                                   {tc_group_properties,
                                                                    []},
                                                                   {tc_group_path,
                                                                    []},
                                                                   {data_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                   {priv_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"}]) failed!
	Reason: {'EXIT',{undef,{ct_test_support,random_error,[],[]}}}

=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:init_per_group
=logfile       config_func_error_1_suite.init_per_group.html
=group_props   [{name,g1}]
=started       2024-10-16 11:52:01
=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_iptc
=logfile       config_func_error_1_suite.exit_in_iptc.5698.html
=started       2024-10-16 11:52:01
=== ERROR! config_func_error_1_SUITE:init_per_testcase(exit_in_iptc, [{watchdog,
                                                                   <0.3743.0>},
                                                                  {tc_logfile,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/config_func_error_1_suite.exit_in_iptc.5698.html"},
                                                                  {tc_group_properties,
                                                                   [{name,
                                                                     g1}]},
                                                                  {tc_group_path,
                                                                   []},
                                                                  {data_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                  {priv_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"}]) failed!
	Reason: {'EXIT',
                    {badarg,
                        {erlang,size,
                            ["/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"],
                            [{error_info,#{module => erl_erts_errors}}]}}}

=ended         2024-10-16 11:52:01
=result        auto_skipped: {failed,
                              {config_func_error_1_SUITE,init_per_testcase,
                               {'EXIT',
                                {badarg,
                                 {erlang,size,
                                  ["/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"],
                                  [{error_info,
                                    #{module => erl_erts_errors}}]}}}}}
=elapsed       0.000000s
=== *** SKIPPED test case 3 of 4 ***
=case          config_func_error_1_SUITE:end_per_group
=logfile       config_func_error_1_suite.end_per_group.html
=group_props   [{name,g1}]
=started       2024-10-16 11:52:01
=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=group_time    0.059s
=case          config_func_error_1_SUITE:init_per_group
=logfile       config_func_error_1_suite.init_per_group.2723.html
=group_props   [{name,g2}]
=started       2024-10-16 11:52:01
=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_eptc
=logfile       config_func_error_1_suite.exit_in_eptc.2755.html
=started       2024-10-16 11:52:01
=== WARNING! config_func_error_1_SUITE:end_per_testcase(exit_in_eptc, [{tc_status,
                                                                    ok},
                                                                   {watchdog,
                                                                    <0.3752.0>},
                                                                   {tc_logfile,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/config_func_error_1_suite.exit_in_eptc.2755.html"},
                                                                   {tc_group_properties,
                                                                    [{name,
                                                                      g2}]},
                                                                   {tc_group_path,
                                                                    []},
                                                                   {data_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                   {priv_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@a46a33ecc1a5.2024-10-16_11.40.24/make_test_dir.common_test_test.logs/run.2024-10-16_11.40.27/log_private/ct_run.ct@a46a33ecc1a5.2024-10-16_11.52.00/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-16_11.52.00/log_private/"}]) failed!
	Reason: {'EXIT',
                    {badarg,
                        {erlang,send,
                            [undef_proc,hello],
                            [{error_info,#{module => erl_erts_errors}}]}}}

=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:end_per_group
=logfile       config_func_error_1_suite.end_per_group.2819.html
=group_props   [{name,g2}]
=started       2024-10-16 11:52:01
=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=group_time    0.059s
=case          config_func_error_1_SUITE:end_per_suite
=logfile       config_func_error_1_suite.end_per_suite.html
=started       2024-10-16 11:52:01
=ended         2024-10-16 11:52:01
=result        ok
=elapsed       0.000000s
=group_time    0.267s
=== TEST COMPLETE, 2 ok, 0 failed, 2 skipped of 4 test cases

=finished      2024-10-16 11:52:01
=failed        0
=successful    2
=user_skipped  0
=auto_skipped  2
=elapsed_time  326381
